docs/support/tutorials/nomachine-usage.md

```diff
@@ -0,0 +1,81 @@
+# How to configure and use the NoMachine remote desktop to access CSC servers
+
+This document will briefly highlight benefits of using NoMachine, how to configure
+connection to CSC and how to further tailor the settings.
+
+[TOC]
+
+A short video will appear in here.
+
+## Why use NoMachine at all?
+
+**Pros:**
+
+- Enables remote grahpics from CSC servers for Windows users
+- Improved remote graphics performance for Linux and Mac users
+- Suspendable session - continue later with all terminals etc. as they were, even from a different computer
+- Easy way to leave jobs running e.g. on Taito-shell while logging off from laptop
+- Gives 4 virtual screens
+- Resolution/frame rate can be tuned - enables usage with even a really slow network
+
+**Cons:**
+
+- Very heavy 3D remote graphics performance will be poor. In this case, consider GPU rendering (link)
+- Requires installing the client (ssh + X11 forwarding are available on Mac & Linux anyway)
+
+## Installation
+
+!!! note 
+    this may require admin privileges
+
+Download the NoMachine Enterprise _client_ (not the Desktop) from here: [https://www.nomachine.com/download-enterprise](https://www.nomachine.com/download-enterprise)
+
+## Configuration
+
+1.   Open NoMachine and click "new" at the top bar (or "Create a new connection" if you see one).
+2.   Accept the "NX" protocol and click continue.
+3.   Fill in nxkajaani.csc.fi as the host, keep the port default port (4000) and click continue.
+4.   Accept the default (password) and click continue.
+5.   Unless you know that you need a proxy, accept the default (no proxy) and click continue. If your organization uses a proxy, contact your local IT support for the configuration.
+6    Give some name for your setup and click Done.
+
+## Open connection
+
+1.   Double click the icon of the connection you just configured.
+2.   Fill in your CSC username and password. 
+3.   After a while, you'll be shown a few pages presenting the different functionality of NoMachine.
+    In the first screen, untick the box from "Change server resolution to match the client when I reconnect".
+4.   Take a while to read through, and if you don't want to see them again, click the box at lower right corner, proceed with "ok".
+5.   Once you see a black screen, you're there. Now right click at the black background and select "CSC local servers" then "Taito".
+6.   You'll be prompted for your CSC password. Give it, and you'll have a terminal open on Taito. The terminal can display remote graphics.
+
+## Recommended tips
+
+1.   To get rid of the scroll bars at the edges of your screen take the mouse to the upper right corner (peel menu) and click "resize remote screen".
+2.   To improve cognitive ergonomy, start a `gnome-terminal` in the fluxbox terminal as the first thing, once logged in Taito/Puhti/... It looks nicer and you can change font size (ctrl-+)
```

Comment:
`gnome-terminal` does not seem to be available on Puhti, at least not with the default modules.